app/src/main/java/nu/yona/app/utils/AppConstant.java

```diff
@@ -292,4 +292,5 @@
 	String RESTART_VPN = "com.yona.app.RESTART_VPN";
 	String RESTART_DEVICE = "com.yona.app.RESTART_DEVICE";
 	String WAKE_UP = "com.yona.app.WAKE_UP_ALARM";
+	String SHOW_VPN_WINDOW = "showVpnWindow";
```

Comment:
Let's stick with the naming convention:
```
String SHOW_VPN_WINDOW = "com.yona.app.SHOW_VPN_WINDOW";
```

The naming convention might be unnecessary, but it's a pattern we have now and it doesn't cause any pain.

Reply:
the pattern is only for broad cast actions, if you see the constants above the visible lines here, there is no pattern. I have placed in the wrong location in the file.

Reply:
OK, fine. Can you move it to a more appropriate place, so that question won't be asked by the next reviewer/editor?